intranet/apps/preferences/fields.py

```diff
@@ -25,7 +22,5 @@ def to_python(self, value):
         return value
 
     def widget_attrs(self, widget):
-        attrs = {}
-        if self.max_length is not None:
-            attrs.update({'maxlength': str(self.max_length)})
+        attrs = {"maxlength": "17"}
```

Comment:
For some reason LDAP is converting the None to a string (which is supposed to happen) as seen [here](https://github.com/tjcsl/ion/blob/master/intranet/apps/preferences/views.py#L97-L102) only for mobile_phone but not any others.